src/locks.ts

```diff
@@ -19,11 +19,11 @@ export async function isLocked(
   )
   const imagesRaw = await runKubectl(kubernetesContext, [
     'get',
-    'pods',
+    'deploy',
```

Comment:
Would there be a drawback of any kind by checking for deployments instead of pods? If there are terminating pods of a previous ReplicaSet, this approach would not cause any more troubles 🤔